tests/_support/Step/Acceptance/Accounts.php

```diff
@@ -1,56 +0,0 @@
-<?php
```

Comment:
Why change the naming convention here? should we update the rest of the acceptance tests to match?

Reply:
Because the Account testers are not Accounts and oposite. rest of tests are refactored.